configs/desktop_savant_events_schemas.json

```diff
@@ -1,5 +1,6 @@
 {
   "source": "telemetry-cohorts",
+  "includePingLevelEvent": true,
```

Comment:
I hate to do this, but I have two suggestions:
- We can make this an event we filter on just like the other events, where category/object/method would probably be something like "meta/telemetry/sessionSplit", and we can stuff that into the ping's `events` value, and we can avoid adding another construct to the schema.
- Alternately if you want to skip that I think this would be more accurately named "includeSessionSplitEvent"

Reply:
includeSessionSplitEvent indeed is a better descriptor here. I do see that the value in making this more like other events, but the top-level field seems more explicit to me. Inserting an event in the stream and then expecting schema writers to know about its coordinates if they want to include it seems more magical. I'm planning to keep it so the `eventGroups` only needs to worry about events defined in the normal way, and the session split ping is a separate path.

Either way we go, we'd end up duplicating a certain amount of code.